src/Maestro/Maestro.Data/Models/Build.cs

```diff
@@ -35,6 +35,18 @@ static Build()
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
 
+        public string Account { get; set; }
```

Comment:
"Account" and "Project" are also AzDo things... And we probably want to spell it out as "AzureDevOpsAccount". Let's not encode this abbreviation in code (especially public code), since I don't think Azure really likes this abbreviation.

Reply:
I agree with Chad. All of these properties should be `AzureDevOps<thingy>`. When you change them make sure to recreate the migration as well. Just revert all your changes to the Migrations folder and make it again.